packages/bus-merge-binding-factory/lib/ActorMergeBindingFactory.ts

```diff
@@ -0,0 +1,45 @@
+import type { IAction, IActorArgs, IActorOutput, IActorTest, Mediate } from '@comunica/core';
+import { Actor } from '@comunica/core';
+import type { IActionContext } from '@comunica/types';
+
+/**
+ * A comunica actor for the creation of merge handlers for binding context keys.
+ *
+ * Actor types:
+ * * Input:  IActionMergeBindingFactory:      TODO: fill in.
+ * * Test:   <none>
+ * * Output: IActorMergeBindingFactoryOutput: TODO: fill in.
+ *
+ * @see IActionMergeBindingFactory
+ * @see IActorMergeBindingFactoryOutput
+ */
+
+export abstract class ActorMergeBindingFactory extends Actor<IActionMergeBindingFactory,
```

Comment:
I know this is an annoying request, but I suggest renaming the bus to `bus-merge-bindings-context`